lib/segment/src/vector_storage/multi_dense/appendable_mmap_multi_dense_vector_storage.rs

```diff
@@ -181,6 +181,15 @@ impl<T: PrimitiveVectorElement> VectorStorage for AppendableMmapMultiDenseVector
         self.offsets.len()
     }
 
+    fn available_size_in_bytes(&self) -> usize {
+        let total_size = self.vectors.len() * self.vector_dim() * std::mem::size_of::<T>();
+        if self.total_vector_count() > 0 {
+            total_size * self.available_vector_count() / self.total_vector_count()
```

Comment:
why don't we just do `self.available_vector_count() * self.vector_dim() * std::mem::size_of::<T>()` ?

Reply:
We cannot do it for multivectors. Your proposal will work only if all multivectors have 1 vector inside. We estimate here available vectors size as a proportion. We guess here that average vector size between existing and deleted vectors are the same

Reply:
Your true, fixed it after already for all multivector storages
https://github.com/qdrant/qdrant/pull/4312/commits/f8400e223b2a0efd0d9744fa91e9f3a8d149e5cb